### DISCUSSION TITLE
Too much miner fee was spend

### DISCUSSION THREAD

**Discussion by MMNNMMNM** [filecoin-project/lotus#9909] [Storage Provider]
### Checklist

- [X] This is **not** a security-related bug/issue. If it is, please follow please follow the [security policy](https://github.com/filecoin-project/lotus/security/policy).
- [X] This is **not** a question or a support request. If you have any lotus related questions, please ask in the [lotus forum](https://github.com/filecoin-project/lotus/discussions).
- [X] This is **not** a new feature request. If it is, please file a [feature request](https://github.com/filecoin-project/lotus/issues/new?assignees=&labels=need%2Ftriage%2Ckind%2Ffeature&template=feature_request.yml) instead.
- [X] This is **not** an enhancement request. If it is, please file a [improvement suggestion](https://github.com/filecoin-project/lotus/issues/new?assignees=&labels=need%2Ftriage%2Ckind%2Fenhancement&template=enhancement.yml) instead.
- [X] I **have** searched on the [issue tracker](https://github.com/filecoin-project/lotus/issues) and the [lotus forum](https://github.com/filecoin-project/lotus/discussions), and there is no existing related issue or discussion.
- [X] I am running the [`Latest release`](https://github.com/filecoin-project/lotus/releases), or the most recent RC(release canadiate) for the upcoming release or the dev branch(master), or have an issue updating to any of these.
- [X] I did not make any code changes to lotus.

### Lotus component

- [ ] lotus daemon - chain sync
- [ ] lotus miner - mining and block production
- [ ] lotus miner/worker - sealing
- [ ] lotus miner - proving(WindowPoSt)
- [ ] lotus miner/market - storage deal
- [ ] lotus miner/market - retrieval deal
- [ ] lotus miner/market - data transfer
- [ ] lotus client
- [ ] lotus JSON-RPC API
- [X] lotus message management (mpool)
- [ ] Other

### Lotus Version

```text
Daemon:  1.18.1+mainnet+git.e68b8e96c+api1.5.0
Local: lotus version 1.18.1+mainnet+git.e68b8e96c
```

### Describe the Bug

A large amount of FIL was withdrawn from our miners as precommit, provecommit, and windowpost miner fees.

This is the Fee content in the config.toml we have.
We've been using it all along and never had this problem until now.

```
[Fees]
  # type: types.FIL
  # env var: LOTUS_FEES_MAXPRECOMMITGASFEE
  MaxPreCommitGasFee = "0.5 FIL"

  # type: types.FIL
  # env var: LOTUS_FEES_MAXCOMMITGASFEE
  MaxCommitGasFee = "0.5 FIL"

  # type: types.FIL
  # env var: LOTUS_FEES_MAXTERMINATEGASFEE
  MaxTerminateGasFee = "0.5 FIL"

  # WindowPoSt is a high-value operation, so the default fee should be high.
  #
  # type: types.FIL
  # env var: LOTUS_FEES_MAXWINDOWPOSTGASFEE
  MaxWindowPoStGasFee = "50 FIL"

  # type: types.FIL
  # env var: LOTUS_FEES_MAXPUBLISHDEALSFEE
  MaxPublishDealsFee = "5 FIL"

  # type: types.FIL
  # env var: LOTUS_FEES_MAXMARKETBALANCEADDFEE
  MaxMarketBalanceAddFee = "0.7 FIL"

  [Fees.MaxPreCommitBatchGasFee]
    # type: types.FIL
    # env var: LOTUS_FEES_MAXPRECOMMITBATCHGASFEE_BASE
    #Base = "0 FIL"

    # type: types.FIL
    # env var: LOTUS_FEES_MAXPRECOMMITBATCHGASFEE_PERSECTOR
    #PerSector = "0.02 FIL"

  [Fees.MaxCommitBatchGasFee]
    # type: types.FIL
    # env var: LOTUS_FEES_MAXCOMMITBATCHGASFEE_BASE
    #Base = "0 FIL"

    # type: types.FIL
    # env var: LOTUS_FEES_MAXCOMMITBATCHGASFEE_PERSECTOR
    #PerSector = "0.03 FIL"
```

   The above is the config.toml file of the miner. According to the contents above, more than 1000FIL should not be used to transfer one sector message.

This is the link of the **slack thread**. It is going to help you to understand the problem.
https://filecoinproject.slack.com/archives/CPFTWMY7N/p1670990322095369

And we attach the capture image of the filfox date including the message.

![miner fee](https://user-images.githubusercontent.com/109844632/208409206-7126786b-4b77-4690-afe9-0d02047b53c9.PNG)
![miner-fee](https://user-images.githubusercontent.com/109844632/208409684-280ce37d-a1e8-4217-97cd-dd7e4ad0b32a.PNG)

### Logging Information

```text
notthing
```

### Repo Steps

1. Run '...'
2. Do '...'
3. See error '...'
...

**Comment by TippyFlitsUK** [comment]
As Patrick mentions in the Slack thread @MMNNMMNM, it appears that the `replace` command has been used. Can you please check your system for automated `replace` scripts?

**Reply by MMNNMMNM** [reply]
We've never used the lotus mpool replace command.
I've never adjusted my gas bill using the CLI.
I have never adjusted the gas fee, but I don't know why a large amount of miner fee was consumed in two miners. 

Please help, we are in a panic.

**Comment by rjan90** [comment]
Hey @MMNNMMNM! First of all I understand how devastating it must feel to transfer that amount of FIL when it was not the intent. Secondly, I´m very glad to see our community being so understanding and helpful that some of the [funds has been returned to you. ](https://filfox.info/zh/message/bafy2bzacecpwcd6mb42hmdgwfilmlx7syuh2bi3sxt434ryxymwqwhp37btyk).

Now as has been repeated in the Slack-thread, and by @TippyFlitsUK here, it seem like the message has been replaced with a script/manually/through custom code. As an example:

- The first ProveCommitSector-message for sector `11255` here: https://filfox.info/en/message/bafy2bzaceb6nix5v4t6qjogeki75u6ze2qt6sbpk3cbwsixhsneqhty64dg6q, has totally appropriate  estimations and values.
- In the same block there is an replaced ProveCommitSector-message for the exact same sector `11255`: https://filfox.info/en/message/bafy2bzaceapwme3ypdnn6fv5ri3ueyzcol5mrdvnvjuzikyfoe7rwadf6ebm2, which has very wrong values set causing the minerFee.

**Reply by MMNNMMNM** [reply]
Hi @rjan90 
Our other messages had similar gas premiums.
But I don't know why only some messages consumed a lot of miner fee.
https://filfox.info/en/message/bafy2bzacecnfjfrtf3vtjces7vdlp37dlr3kbov2vw2rhgoh7mwlgprb35kn6

**Reply by rjan90** [reply]
This message has a very different gas-premium compared to the message with the high minerFee. The confusion might be around the [prefixes](https://en.wikipedia.org/wiki/International_System_of_Units)?

For the [...rb35kn6](https://filfox.info/en/message/bafy2bzacecnfjfrtf3vtjces7vdlp37dlr3kbov2vw2rhgoh7mwlgprb35kn6) messageID with low minerFee:
- Gas Premium = 100,695 attoFIL = 0.000000000000101 FIL

While the [...6ebm2](https://filfox.info/en/message/bafy2bzaceapwme3ypdnn6fv5ri3ueyzcol5mrdvnvjuzikyfoe7rwadf6ebm2) messageID with a high minerFee:
- Gas Premium = 0.000021113 FIL

**Comment by MMNNMMNM** [comment]
These are the sealing script commands we use.
We have never changed the gas cost part, but that kind of phenomenon has occurred.

```
#!/bin/bash

number=0

while [ $number -le 12 ]
#while true
do
  echo "Number: ${number}"
  ((number++))
  lotus-miner sectors pledge
  sleep 100
done
```

I'll say it again, but we're really having a hard time
We managed Miner as usual.
But the results are very different.
please help us keep miner we want to go with filecoin
